### ISSUE TITLE
GE LIST calculator isn't working

### ISSUE THREAD

**Issue by npetheriot** [Joshua-Chiu/PGDBWebServer#25]
Sending on behalf of Miss Chan...because she deals with honour rolls...the honour roll list works...the principal's list works...but the GE list produces nothing.  So, in order for this to work, we need the calculator here to scan all report cards and only include students with G and/or E as work habits on their report cards.  So, if they have an S or N in any subject, they should not be on the list.  Merci! :)

**Comment by mason-anderson** [comment]
Josh sent me the file and the problem is that the top row with the key is missing. The program relies on column names at the top to know where to look. This might be fixed in the future, but for now try to include the key.

**Comment by npetheriot** [comment]
Thank you and now I will get Josh to explain that in plain English please?  🤣

Sent from my iPhone

On Apr 29, 2020, at 6:44 PM, Mason Anderson <notifications@github.com> wrote:

﻿

Josh sent me the file and the problem is that the top row with the key is missing. The program relies on column names at the top to know where to look. This might be fixed in the future, but for now try to include the key.

**Comment by Joshua-Chiu** [comment]
@mason-anderson Make the pgdb file export GE standings too as stated in the issue.
